src/containers/layout/language-menu/LanguageMenu.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const languageMenuConstants = [
+  {
+    languageCode: 'en',
+    language: 'English'
+  },
+  {
+    languageCode: 'ua',
+    language: 'Українська'
+  }
+]
```

Comment:
In such cases, it is better to name keys as `{value: '...', label: '...'}`.

Reply:
Changed